tests/components/application-connector/test/application-gateway/runner_test.go

```diff
@@ -71,3 +73,32 @@ func (gs *GatewaySuite) TestResponseBody() {
 		})
 	}
 }
+
+func (gs *GatewaySuite) TestBodyPerMethod() {
+	app, err := gs.cli.ApplicationconnectorV1alpha1().Applications().Get(context.Background(), "body-forwarding", v1.GetOptions{})
+	gs.Nil(err)
+	for _, service := range app.Spec.Services {
+		gs.Run(service.Description, func() {
+			httpCli := NewHttpCli(gs.T())
+
+			for _, entry := range service.Entries {
+				if entry.Type != "API" {
+					gs.T().Log("Skipping event entry")
+					continue
+				}
+
+				req, err := http.NewRequest(service.Description, entry.CentralGatewayUrl, strings.NewReader(service.Description))
```

Comment:
I'd explicitly save these in some variables to make it more readable. You know, `method := service.Description` and `body := strings.NewReader(service.Description)`. It'd be more clear that we use this description as we do